client.go

```diff
@@ -43,9 +42,18 @@ var (
 	DefaultNotifyBufSize = 5
 )
 
+// Connection is a connection to a TeamSpeak 3 server.
+// It a wrapper around net.Conn with a Connect method.
+type Connection interface {
+	net.Conn
+
+	// Connect connects to the server on addr and given timeout.
```

Comment:
Slight improvement because it doesn't connect on a timeout.
```suggestion
	// Connect connects to the server on addr with a timeout.
```